data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json

```
@@ -69,7 +69,6 @@
     "apply_message": "You spent some pleasant time relaxing with someone.",
     "miss_messages": [ [ "You feel relaxed.", 1 ] ],
     "rating": "good",
```

```suggestion
    "rating": "good",
    "cures_effects": [ "bad_food_ennui", "sad" ]
```
Morale modifications by effects are unfortunately currently not possible, as far as I am aware. (I don't know whether having "sad" above will work, given its relationship with morale.)

---

@actual-nh The game throws an error when I try adding those two. Presumably morale effects are coming, maybe we should just wait for those?

---

Was that before or after removing the comma from after `"good"`?

---

@actual-nh The comma is there. It says Invalid or misplaced field name "cures_effects" in JSON data

    "rating": "good",
    "cures_effects":
                             ^ [ "bad_food_ennui", "sad" ]
  },
  {

---

Oops! I was looking at the wrong thing - should be "removes_effects"... sorry!

---

Now it loads but it doesn't remove the Sad effect. I'm not sure it should affect anything else, there are a million other mood debuffs this doesn't cure and it seems weird to just have it doing one for eating bland food and nothing else.

I think we should wait for better morale effects before adding anything to Alonso here.

---

Sigh... OK. There is a note with the bland food effect:
`"//": "No morale_mod is currently possible in effects, for some reason.  As soon as it's implemented, it goes here."`

Perhaps put that in here also?

---

Done.